### PR TITLE
fusion360 keyboard shortcut

### PR DESCRIPTION
**Pull request by salmanfarisvp**
## Description of new Instant Answer, or changes
fusion360 keyboard shortcut

## Related Issues and Discussions
Source: https://www.cadlinecommunity.co.uk/hc/en-us/articles/210361725-Fusion-360-Keyboard-Shortcuts

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/fusion360_keyboard_shortcuts

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/Fusion360.json [hunk 1]
```diff
@@ -0,0 +1,349 @@
+{
+    "id": "fusion360_keyboard_shortcuts",
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/Fusion360.json]: id should end with  `_cheat_sheet`

File: share/goodie/cheat_sheets/json/Fusion360.json [hunk 2]
```diff
@@ -0,0 +1,349 @@
+{
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/Fusion360.json]: File name should reflect ID.

    -> FAIL: Invalid file name (Fusion360.json) for ID (fusion360_keyboard_shortcuts)

**Comment by kirkins** [comment]
@salmanfarisvp it's good your doing this but I think it's better to make an update to PR rather than close and making a new one. That way you don't spam the PR list.

**Pull request closed without merging**